bin/selbot2.rb

```diff
@@ -208,6 +208,11 @@
       :expression => /┻━*┻/,
       :text       => "┬──┬ ノ( ゜-゜ノ)",
       :help       => "unflips the table"
+    },
+    {
+      :expression => /:blame/,
+      :text => "Saying \"It's broken\" isn't helpful to anyone, especially us - if you have a specific issue, please detail it and submit it to the appropriate project's bug tracker with a reproduceable test case",
+      :help => "It's broken is pointless"
```

Comment:
I feel like this may come off a little harsh... How about...

```
Saying "It's broken" is nondescript. If you have a question, please ask with details. If you have a reproducible test case, please submit to the appropriate project's bug tracker.
```

Help:
```
provide more details for broken claim
```

Also maybe change the expression to `:broken` ?